From ede2307de277a896be281ad8d0de004a97c61f5a Mon Sep 17 00:00:00 2001
From: "Alexander G. M. Smith" <agmsmith@ncf.ca>
Date: Thu, 11 Nov 2021 18:37:54 -0500
Subject: [PATCH] BeShare: Hack port numbers to start at 7050 instead of 7000.

Useful if you want to run two BeShares and not have them interfere.
---
 source/beshare/ShareConstants.h | 2 +-
 source/beshare/ShareWindow.cpp  | 3 ++-
 2 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/source/beshare/ShareConstants.h b/source/beshare/ShareConstants.h
index 6259741..8fd8013 100644
--- a/source/beshare/ShareConstants.h
+++ b/source/beshare/ShareConstants.h
@@ -8,7 +8,7 @@ namespace beshare {
 #define VERSION_STRING "3.04"
 #define VERSION_DATE "2018/11/07"
 
-#define DEFAULT_LISTEN_PORT 7000
+#define DEFAULT_LISTEN_PORT 7050
 #define LISTEN_PORT_RANGE   50
 #define CLUMP_CHAR          '\1'
 
diff --git a/source/beshare/ShareWindow.cpp b/source/beshare/ShareWindow.cpp
index c686eee..11e8395 100644
--- a/source/beshare/ShareWindow.cpp
+++ b/source/beshare/ShareWindow.cpp
@@ -3144,7 +3144,8 @@ void ShareWindow :: MessageReceived(BMessage * msg)
          sprintf(temp, "BeShare v%s\n"
             "%s Jeremy Friesner\njaf@lcsaudio.com\n\n"
             "%s Vitaliy Mikitchenko\n"
-            "Updates since 2.28: BBJimmy, Pete, AGMS et al.",
+            "Updates since 2.28: BBJimmy, Pete, AGMS et al.\n"
+            "Hacked by AGMS to start ports at 7050 for IRC bridge.",
             VERSION_STRING, str(STR_BY), str(STR_COLOR_PREFS_BY));
          const char * url = NULL;
          switch((new BAlert(str(STR_ABOUT_BESHARE), temp, "Muscle Page", "BeShare Page", "Okay"))->Go())
-- 
2.30.2

